docs/kratos/self-service/flows/account-recovery-password-reset.mdx

```diff
@@ -0,0 +1,237 @@
+           "ory.sh/kratos": {
+             "credentials": {
+               "password": {
+                 "identifier": true
+               }
+             },
++            "recovery": {
++              "via": "email"
++            }
+           }
+         }
+       }
+       "additionalProperties": false
+     }
+   }
+ }
+```
+
+## Email templates
+
+<p>
+  <figure>
+    <img alt="Recovery email sent to unknown address" src={useBaseUrl("images/kratos/email-recovery-unknown.png")} />
+    <figcaption>
+      If the requested email address is an unknown recovery address, an account recovery attempt email is sent to that email
+      address.
+    </figcaption>
+  </figure>
+</p>
+
+This prevents account enumeration attacks as explained in this
+[blog post by Troy Hunt](https://www.troyhunt.com/website-enumeration-insanity-how-our-personal-data-is-leaked/).
+
+<p>
+  <figure>
+    <img alt="Recovery email sent to a known recovery address" src={useBaseUrl("images/kratos/email-recovery-known.png")} />
+    <figcaption>
+      If the requested email address is a known recovery address, a recovery code is sent to that email address.
+    </figcaption>
+  </figure>
+</p>
+
+The emails are using templates that can be customized as explained in
+[Customizing E-Mail Templates](../../emails-sms/custom-email-templates).
+
+## Updating credentials
+
+After the user completes the recovery successfully they receive a privileged session and are redirected to the settings page to
+update their credentials.
+
+Read more about the [privileged sessions](../../session-management/01_overview.mdx#privileged-sessions).
+
+## Invalidate other sessions
+
+In some scenarios it can be useful to revoke all active sessions once an account has been recovered. This forces anyone with
+access to the account to re-authenticate.
+
+```diff
+ selfservice:
+   flows:
+     recovery:
+       enabled: true
+       ui_url: http://127.0.0.1:4455/recovery
++      after:
++        hooks:
++        - hook: revoke_active_sessions
+```
+
+## One-time passwords vs. magic links
+
+The one-time password method (`code`) is preferred, as magic links have multiple drawbacks:
+
+- Some email virus scanners open links in emails to scan them. This invalidates the magic link and a user may not be able to
+  recover their account.
+- API flows initialized by apps on mobile phones or smart devices are not possible when using magic links.
+- Magic links often open in different browsers than the one used to initialize the flow, depending on the user's device settings.
+  This may result in a confusing user experience for your users.
+
+Ory Identities (Ory Kratos) uses one-time passwords by default. Magic links are available for legacy reasons and will be removed
+in the future.
+
+:::important
+
+When updating an existing project from the `link` method to use the `code` method, there might be changes in the behavior of the
+UI. Double-check the flow after updating your configuration.
```

Comment:
```suggestion
When you change the recovery strategy from `link` to `code` in an existing project, you might need to adjust the UI to accommodate for that flow. Make sure the flow works correctly with your UI implementation after changing the recovery strategy.
```
Nice one, lemme one-up it ;)

Reply:
Lot of "flow"s in there...

I think there was some talk, that we want to remove the word "flow", as it's more of a technical term used internally in the codebase. So I've adjusted your proposal a bit more.